examples/bri-3/src/bri/decorators/public-endpoint.ts

```diff
@@ -1,4 +1,4 @@
 import { SetMetadata } from '@nestjs/common';
 
-export const IS_PUBLIC_KEY = 'isPublic';
-export const Public = () => SetMetadata(IS_PUBLIC_KEY, true);
+export const IS_PUBLIC_ENDPOINT_KEY = 'isPublic';
```

Comment:
i guess we can rename this to IS_PUBLIC_ENDPOINT

Reply:
I could take it either way, but nestjs references it as the "metadata key"